packages/nx-plugin/src/generators/package/nest/app/files/vite.config.ts__tmpl__

```diff
@@ -0,0 +1,13 @@
+import getConfig from '@eternagame/vite-utils';
+import nestSwaggerPlugin from '@nestjs/swagger/plugin.js';
+
+export default getConfig({
+  type: 'app',
+  env: 'node',
+  tsTransformers: [
```

Comment:
What is this tsTransformer option for? Is this to address some of the Swagger decorators?

Reply:
See https://docs.nestjs.com/openapi/cli-plugin#cli-plugin - it basically lets us take some shortcuts when defining DTOs (ie, classes which represent request bodies) and shortcutting some boilerplate for exposing parameters to OpenAPI

Reply:
(Transforms are basically a form of compiler plugin for typescript - though this is the only use-case I currently have for them)